feather/server/src/packet_handlers/inventory.rs

```diff
@@ -54,7 +54,7 @@ pub fn handle_click_window(
     if packet.slot >= 0 {
         client.set_slot(packet.slot, window.item(packet.slot as usize)?.clone());
     }
-    client.set_cursor_slot(window.cursor_item());
+    client.set_cursor_slot(window.cursor_item().clone());
```

Comment:
Safety comment... can we safely do this without introducing a duplication glitch?

Reply:
Ownership was taken because packets always takes ownership. The value was never stored anywhere so it was safe. Moved the clone closer to the packet creation such that it was made more obvious that it is safe.